src/Data/Integer/Properties.agda

```diff
@@ -14,6 +14,7 @@ module Data.Integer.Properties where
 open import Algebra.Bundles
 import Algebra.Morphism as Morphism
 import Algebra.Properties.AbelianGroup
+open import Data.Empty using (⊥-elim)
```

Comment:
We tend to try and use `contradiction` from `Relation.Nullary.Negation` rather than `⊥-elim` as it makes the code more readable.